From 5b825dec7956026bae5d5f3ea0e4d2196e82a9a5 Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Tue, 4 Aug 2015 15:10:40 +0800
Subject: [PATCH] Ignore the default include dir -I/usr/include

Upstream-status: Inappropriate [embedded specific]

The output from `icu-config --cppflags-searchpath` is "-I/usr/include" and will
lead to host contamination. Just ignore it since that's the default path.

Signed-off-by: Jesse Zhang <sen.zhang@windriver.com>

This patch fix configure error:

  ERROR: This autoconf log indicates errors, it looked at host include \
    and/or library paths while determining system capabilities.
  ERROR: Function failed: do_qa_configure

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
---
 m4/xerces_icu_prefix.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/xerces_icu_prefix.m4 b/m4/xerces_icu_prefix.m4
index 5c50754..2e6da25 100644
--- a/m4/xerces_icu_prefix.m4
+++ b/m4/xerces_icu_prefix.m4
@@ -38,7 +38,7 @@ AC_DEFUN([XERCES_ICU_PREFIX],
                   # can use them uniformly whether they are empty or not.
                   #
                   if test -n "$icu_config"; then
-                    icu_flags=`$icu_config --cppflags-searchpath`
+                    icu_flags=`$icu_config --cppflags-searchpath | sed s:-I/usr/include::`
                     icu_libs=`$icu_config --ldflags`
                     icu_bin=`$icu_config --bindir`
                     icu_sbin=`$icu_config --sbindir`
-- 
1.9.1

